packages/audio-block/src/AudioBlock.tsx

```diff
@@ -86,7 +89,7 @@ export const AudioBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
         }
     };
 
-    const onTitleChange = (title: string) => title !== blockSettings.title && setBlockSettings({ title });
+    const onTitleChange = async (title: string) => title !== blockSettings.title && (await setBlockSettings({ title }));
```

Comment:
```suggestion
    const onTitleChange = async (title: string) => {
    if(title !== blockSettings.title){
    await setBlockSettings({ title });
    }}
```
Otherwise the typing on the return of this function ends up being false | Promise<void> which feels a bit off

Reply:
True didn't recognized it